upload/upload.go

```diff
@@ -191,6 +191,13 @@ func NewHandler(
 
 		validator.Validate(vr)
 
-		w.WriteHeader(http.StatusAccepted)
+		metadata, err := readMetadataPart(r)
+		if metadata != nil {
+			w.WriteHeader(http.StatusAccepted)
+		} else {
+			w.WriteHeader(http.StatusCreated)
+			w.Write([]byte("{}"))
```

Comment:
Should probably write this out in every case.  Is there anything useful we can return?